layer/trace_layer_with_xr.def

```diff
@@ -0,0 +1,10 @@
+LIBRARY VkLayer_gfxreconstruct
+EXPORTS
+vkGetInstanceProcAddr
+vkGetDeviceProcAddr
+vkEnumerateInstanceLayerProperties
+vkEnumerateDeviceLayerProperties
+vkEnumerateInstanceExtensionProperties
+vkEnumerateDeviceExtensionProperties
+vkNegotiateLoaderLayerInterfaceVersion
+xrNegotiateLoaderApiLayerInterface
```

Comment:
I think I always expose these (just as stubs) so we can share the same def/map files.

Reply:
Don't really want to change this at this late state... with all the build permutation testing.